FSQLocationBrokerTests/FSQTestingHelper.h

```diff
@@ -0,0 +1,47 @@
+//
+//  FSQTestingHelper.h
+//  FSQLocationBroker
+//
+//  Created by Eric Bueno on 4/18/17.
+//  Copyright © 2017 Foursquare. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+#import <CoreLocation/CoreLocation.h>
+
+#pragma mark Date Equality Macros
+
+#define FSQAssertEqualDates(date1, date2) \
```

Comment:
Do these need to be macros or can they be functions? I always prefer functions whenever possible, much easier to debug and harder to shoot yourself in the foot.